packages/common/src/schema.graphql

```diff
@@ -925,6 +925,9 @@ input UpdateInboundShipmentInput {
   onHold: Boolean
   comment: String
   theirReference: String
+
+  ## Additions
+  color: String
```

Comment:
I added this to test the update was working, but I want to remove it when the backend is working again.. or maybe I won't need to?

Reply:
the `color` was added to develop and working in the front-end-fun branch, should be ok to leave